pkg/trustedcabundle/trustedcabundle.go

```diff
@@ -28,9 +28,7 @@ const (
 )
 
 func ShouldInjectTrustedBundle(ns *corev1.Namespace) bool {
-	isActive := ns.Status.Phase == corev1.NamespaceActive
-
-	return isActive && cluster.IsNotReservedNamespace(ns) && !HasCABundleAnnotationDisabled(ns)
+	return cluster.IsNotReservedNamespace(ns) && !HasCABundleAnnotationDisabled(ns)
```

Comment:
Any reason as to why checking if the ns is active is not relevant anymore?

Reply:
in my last commit, i removed this check, feel it is duplicated if it has been done in ProcessAllNamespace

Reply:
you are right, other places also call ShouldInjectTrustedBundle() so it is needed to have status check.